src/style.css

```
@@ -0,0 +1,139 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  font-family: Avenir, sans-serif;
+  box-sizing: border-box;
+}
+
+.searchBar {
```

```suggestion
.search-bar {
```
use kebab-case for class names